mypy.ini

```diff
@@ -282,6 +282,9 @@ disallow_untyped_defs = True
 [mypy-tests.storage.test_user_directory]
 disallow_untyped_defs = True
 
+[mypy-tests.rest.client.test_directory]
+disallow_untyped_defs = True
```

Comment:
I think this is unnecessary with #11322?

Reply:
OH NO! That had a bug in it....

Reply:
missing a `.*` I think? Ouch!